cypress/e2e/specialFeatures/liteSiteWeight/helpers/getTotalPageSize.js

```diff
@@ -3,19 +3,23 @@ import roundTo2Decimals from './roundTo2Decimals';
 
 export default requests => {
   let totalSize = 0;
+  const requestSizes = [];
+
   // eslint-disable-next-line cypress/unsafe-to-chain-command
   return cy
     .wrap(requests)
     .each(({ url, contentLength }) => {
       if (contentLength) {
+        requestSizes.push({ url, size: contentLength });
```

Comment:
Weirdly not seeing the ATI requests listed in the output (I was getting them locally when we first did the curl / content-length stuff...)

Reply:
Yeaaa , it might be a timing thing? as those are fired off after the page load, so maybe they're not getting fired off fast enough to be captured in the requests sometimes - think maube this is fine as they're not _technically_ part of the inital data a user has to download?